scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -196,110 +183,6 @@ timeit()
     return $extcode
 }
 
-#
-# Configures the commands for encrypt=1
-# Sets up the parameters (algo/keys) for XB-based encryption
-# If the version of PXB is >= 2.4.7, changes the XB parameters
-# If the version of PXB is < 2.4.7, use xbcrypt instead
-#
-get_keys()
-{
-    # $encrypt -eq -1 is for internal purposes only
-    if [[ $encrypt -ge 2 || $encrypt -eq -1 ]]; then
-        return
-    fi
-
-    # TODO: it seems like encrypt=1/2/3/4 was not provided
-    if [[ $encrypt -eq 0 ]]; then
-        if $MY_PRINT_DEFAULTS -c $WSREP_SST_OPT_CONF xtrabackup | grep -q encrypt; then
-            wsrep_log_warning "Encryption configuration mis-match. SST may fail. Refer to http://www.percona.com/doc/percona-xtradb-cluster/manual/xtrabackup_sst.html for more details."
```

Comment:
Are we sure we want to get rid of this message if encryption is disabled?

Reply:
message moved, see below

Reply:
Sorry, but I can't see where we print the message if encrypt == 0. Maybe it is not important anymore, so just consider my comment like a question, not the recommendation.

Reply:
The encrypt setting doesn't matter anymore.  Using the xtrabackup encryption settings will ALWAYS cause an SST failure  (now that we've removed encrypt=1, which is the only scenario where it might have worked).